tmt/steps/provision/__init__.py

```diff
@@ -1103,6 +1109,59 @@ def requires(cls) -> list['tmt.base.Dependency']:
         """ All requirements of the guest implementation """
         return []
 
+    def acquire_log(self, log_name: str) -> Optional[str]:
+        """
+        Fetch and return content of a log.
+        :param log_name: name of the log.
+        :returns: content of the log.
+        """
+        raise NotImplementedError
+
+    def store_log(
+            self,
+            log_path: Path,
+            log_content: str,
+            log_name: Optional[str] = None) -> None:
+        """
+        Save log content to a file.
+        :param log_path: a path to save into,could be a directory
+        or a file path.
```

Comment:
This line needs to be indented by 4 spaces, to be rendered correctly.

Reply:
Also, there should be a space after `into,`.

Reply:
By "this line",you mean "or a file path"? I failed to see the problem,I have ran "pre-commit run --all-files" before I push the code.

Reply:
Yes, I meant "or a file path". It's what Sphinx expects, if one needs to split a lengthy parameter description, lines need to be indented to be rendered properly.

```
:param foo: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
    do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
    enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi
    ut aliquip ex ea commodo consequat.
```